assets/bingo/lang/pl_pl.json

```
@@ -0,0 +1,411 @@
+	"bingo.preferences.custom_hud.adjust_col.move_up.title": "▲",
+	"bingo.preferences.custom_hud.adjust_col.move_up.description": "Zamienia pozycję tego komponentu z komponentem powyżej.",
+	"bingo.preferences.custom_hud.adjust_col.move_down.title": "▼",
+	"bingo.preferences.custom_hud.adjust_col.move_down.description": "Zamienia pozycję tego komponentu z komponentem poniżej.",
+	"bingo.preferences.custom_hud.adjust_col.remove.title": "Usuń",
+	"bingo.preferences.custom_hud.adjust_col.remove.description": "Usuwa komponent z HUD'a.",
+	"bingo.preferences.custom_hud.adjust_col.add_spacer": "Dodaj Odstęp",
+
+	"bingo.preferences.custom_hud.adjust_card.title": "Dostosuj Kartę Bingo",
+	"bingo.preferences.custom_hud.adjust_card.description": "Tutaj możesz zmienić ustawienia Karty Bingo.",
+	"bingo.preferences.custom_hud.adjust_card.left_of_hotbar.title": "Przesuń na Lewo od Paska Narzędzi",
+	"bingo.preferences.custom_hud.adjust_card.left_of_hotbar.description": "Przesuwa kartę Bingo na lewo od paska narzędzi.",
+	"bingo.preferences.custom_hud.adjust_card.right_of_hotbar_no_attack.title": "Przesuń na Prawo od Paska Narzędzi",
+	"bingo.preferences.custom_hud.adjust_card.right_of_hotbar_no_attack.description": "Przesuwa kartę Bingo na prawo od paska narzędzi bez zostawienia miejsca na wskaźnik ataku.\nW przypadku gdy “Ustawienia → Ustawienia graficzne → Wskaźnik ataku” jest ustawiony na “Pasek narzędzi”, proszę wybrać drugą opcję.",
+	"bingo.preferences.custom_hud.adjust_card.right_of_hotbar_with_attack.title": "Przesuń na Prawo od Paska Narzędzi (Wskaźnik Ataku)",
+	"bingo.preferences.custom_hud.adjust_card.right_of_hotbar_with_attack.description": "Przesuwa kartę Bingo na prawo od paska narzędzi zostawiając miejsce na wskaźnik ataku.\nUżywać tylko, gdy “Ustawienia → Ustawienia graficzne → Wskaźnik ataku” jest ustawiony na “Pasek narzędzi”.",
+
+	"bingo.custom_hud.components.spacer": "Odstęp",
+	"bingo.custom_hud.components.seed": "Ziarno Bingo",
+	"bingo.custom_hud.components.player_position": "Współrzędne X/Z",
+	"bingo.custom_hud.components.y_position": "Współrzędna Y",
+	"bingo.custom_hud.components.timer": "Stoper",
+
+	"bingo.lobby.machinery.sign.line1": "Maszynownia",
+	"bingo.lobby.machinery.sign.line2": "",
+	"bingo.lobby.machinery.sign.line3": "Nieupoważnionym",
+	"bingo.lobby.machinery.sign.line4": "wstęp wzbroniony",
+
+	"bingo.lobby.credits.neun_einser.sign.line2": "",
+	"bingo.lobby.credits.neun_einser.sign.line3": "Całość Minecraft",
+	"bingo.lobby.credits.neun_einser.sign.line4": "Bingo v5",
+
+	"bingo.lobby.credits.amber_wat.sign.line2": "Czcionka",
+	"bingo.lobby.credits.amber_wat.sign.line3": "Negative Space",
+	"bingo.lobby.credits.amber_wat.sign.line4": "Font",
+
+	"bingo.lobby.credits.nope_name.sign.line2": "Paczka zasobów",
+	"bingo.lobby.credits.nope_name.sign.line3": "narzędzi",
+	"bingo.lobby.credits.nope_name.sign.line4": "matematycznych",
```

Google translate gave me "resource pack" here, but it is a data pack. I found "Paczki danych" for data pack in the Polish wiki, not sure if it needs to be grammatically adjusted.

---

Yup, don't know how I made that mistake 🤔
```suggestion
	"bingo.lobby.credits.nope_name.sign.line2": "Paczka danych",
	"bingo.lobby.credits.nope_name.sign.line3": "narzędzi",
	"bingo.lobby.credits.nope_name.sign.line4": "matematycznych",
```
_Paczk`i` = multiple packs
Paczk`a` = one pack ;)_